{"schemaVersion":{"minor":3,"major":0,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason\/networktransporterror"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason\/networkTransportError","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"role":"symbol","title":"ReaderConnectionFailureReason.networkTransportError","externalID":"c:@M@MobilePaymentsSDKAPI@E@SQMPReaderConnectionFailureReason@SQMPReaderConnectionFailureReasonNetworkTransportError","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"networkTransportError"}],"modules":[{"name":"MobilePaymentsSDK"}],"symbolKind":"case","roleHeading":"Case"},"hierarchy":{"paths":[["doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI","doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason"]]},"abstract":[{"type":"text","text":"There was a network error while connecting to Square."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"networkTransportError","kind":"identifier"}],"platforms":["iOS"]}]}],"references":{"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI":{"abstract":[],"identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI","url":"\/documentation\/mobilepaymentssdkapi","type":"topic","title":"MobilePaymentsSDK","kind":"symbol","role":"collection"},"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI/ReaderConnectionFailureReason":{"role":"symbol","navigatorTitle":[{"text":"ReaderConnectionFailureReason","kind":"identifier"}],"identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason","url":"\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason","title":"ReaderConnectionFailureReason","abstract":[{"type":"text","text":"Indicates why a reader might have failed to connect to Square."}],"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ReaderConnectionFailureReason","kind":"identifier"}],"kind":"symbol"},"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI/ReaderConnectionFailureReason/networkTransportError":{"url":"\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason\/networktransporterror","title":"ReaderConnectionFailureReason.networkTransportError","identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason\/networkTransportError","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"networkTransportError"}],"type":"topic","kind":"symbol","abstract":[{"type":"text","text":"There was a network error while connecting to Square."}]}}}